Makefile

```diff
@@ -33,6 +40,30 @@ clean:
 	rm -f *~ *.o *.d ext/*/*.o $(PROGRAMS) navmon.pb.h navmon.pb.cc $(patsubst %.cc,%.o,$(wildcard ext/sgp4/libsgp4/*.cc)) $(H2OPP) $(SIMPLESOCKETS)
 	rm -f ext/fmt-5.2.1/src/format.o
 
+help2man:
+	$(INSTALL) -m 755 -d $(DESTDIR)$(prefix)/share/man/man1
+	HELP2MAN_DESCRIPTION=Open-source GNSS Monitoring Project
+	$(foreach binaryfile,$(PROGRAMS),help2man -N -n "$(HELP2MAN_DESCRIPTION)" ./$(binaryfile) | gzip > $(DESTDIR)$(prefix)/share/man/man1/$(binaryfile).1.gz;)
+	@echo until these binaries support --help and --version remove the broken output
+	rm -f $(DESTDIR)$(prefix)/share/man/man1/testrunner.1.gz
+
+install: $(PROGRAMS) help2man
+	$(INSTALL) -m 755 -d $(DESTDIR)$(prefix)/bin
+	$(foreach binaryfile,$(PROGRAMS),$(INSTALL) -s -m 755 -D ./$(binaryfile) $(DESTDIR)$(prefix)/bin/$(binaryfile);)
+	@echo "using cp instead of install because recursive directories of ascii"
+	mkdir -p $(DESTDIR)$(prefix)$(htdocs)/galmon
+	cp -a html $(DESTDIR)$(prefix)$(htdocs)/galmon/
+
+install-debian:
```

Comment:
This confuses me greatly. I'd assume 'make install-debian' would mean "install what you just compiled in reasonable Debian compatible paths" and not "reconfigure apt to get packages from ota.bike".

Reply:
Trying "download-debian-package" as a distinct placeholder.